src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -356,9 +358,11 @@ private Vector limitDirectionSlewRate(Vector spdVec) {
     double inputTranslationDir = spdVec.angle();
     double inputTranslationMag = spdVec.mag();
 
+    SmartDashboard.putNumber("stick ang", inputTranslationDir / Math.PI);
```

Comment:
Make this key a constant in RobotConfig.

Reply:
this is for logging and can be removed

Reply:
You can leave it in. it will be useful for later if we need to debug.